src/task1/task1_2.ts

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs';
+import readline from 'readline';
+import csv from 'csvtojson';
+
+const sourceFilePath = './src/csv/hw1-ex1.csv';
+const destFilePath = './src/task1/files/hw1-ex1.txt';
+const readStream = fs.createReadStream(sourceFilePath);
+const writeStreamFile = fs.createWriteStream(destFilePath, 'utf8');
+
+function writeLine(line: string) {
+  console.log(`Line from file: ${line}`);
```

Comment:
It is not critical for this task, but usually such type of logs written with `console.debug` method. It is usefull on production environment when you don't need detailed information about data processing but also give you ability to turn on showing it with PROCESS_ENV=debug mode.
By the way, logging is very important part in AWS development 🙂

Reply:
Thanks you)